pytorch_lightning/trainer/data_loading.py

```diff
@@ -212,14 +213,19 @@ def _reset_eval_dataloader(self, model: LightningModule, mode: str) -> Tuple[int
             dataloaders = [dataloaders]
 
         # shuffling in val and test set is bad practice
+        dataloadersClean = []
         for loader in dataloaders:
-            if mode in ('val', 'test') and hasattr(loader, 'sampler') and isinstance(loader.sampler, RandomSampler):
-                raise MisconfigurationException(
-                    f'Your {mode}_dataloader has shuffle=True, it is best practice to turn'
-                    ' this off for validation and test dataloaders.')
+            if loader is not None:
+                if mode in ('val', 'test') and hasattr(loader, 'sampler') and isinstance(loader.sampler, RandomSampler):
+                    raise MisconfigurationException(
+                        f'Your {mode}_dataloader has shuffle=True, it is best practice to turn'
+                        ' this off for validation and test dataloaders.')
+                dataloadersClean.append(loader)
```

Comment:
this should be `dataloaders_clean`, i'm surprised our pep8speaks bot didn't mention this 
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names